docs/manual/developer_guide.adoc

```diff
@@ -1,4 +1,4 @@
-= SCAP Security Guide Developer Guide
+= ComplianceAsCode Developer Guide
```

Comment:
wait, ComplianceAsCode is the org, not the project, right?

Reply:
I think we can put it here as well.